src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -808,6 +808,12 @@ public enum Mixins implements IMixins {
             .setApplyIf(() -> TweaksConfig.ic2CellWithContainer)
             .addRequiredMod(TargetedMod.IC2)
             .setPhase(Phase.LATE)),
+    IC2_REACTORS_FIX_LAG(new MixinBuilder("Cache IC2 reactor size to reduce lag")
```

Comment:
there is another mixin that optimizes the IC2 reactors it's `MixinDirection_Memory`, you should put both under the same config